KsApi/Service.swift

```diff
@@ -107,8 +107,10 @@ public struct Service: ServiceType {
   }
 
   public func createBacking(input: CreateBackingInput) ->
-    SignalProducer<CreateBackingEnvelope, GraphError> {
-    return applyMutation(mutation: CreateBackingMutation(input: input))
+    SignalProducer<CreateBackingEnvelope, ErrorEnvelope> {
+    return GraphQL.shared.client
+      .perform(mutation: GraphAPI.CreateBackingMutation(input: GraphAPI.CreateBackingInput.from(input)))
```

Comment:
I guess we can remove dependency on `CreateBackingInput` in the future.

Reply:
Just because it's still relying on the `GraphMutationInput` here.

Reply:
I think it'll be ok to remove the conformance to `GraphMutationInput` and then it'll just be a kind've helper type. But you could also choose to expand the interface for `ServiceType` to just take all of the arguments directly. This was just to make as few changes as possible right now.